tools/python/boutiques/schema/examples/good.json

```diff
@@ -1,475 +1,475 @@
 {
-    "command-line": "ndmg_pipeline [DTI] [BVAL] [BVEC] [MPRAGE] [ATLAS] [MASK] [OUTDIR] [DESIKAN] [JHU] [TALAIRACH] [AAL] [HARVARDOXFORD] [CPAC200] [CLEAN] [EXTRA1] [EXTRA2] [EXTRA3]", 
+    "command-line": "ndmg_pipeline [DTI] [BVAL] [BVEC] [MPRAGE] [ATLAS] [MASK] [OUTDIR] [DESIKAN] [JHU] [TALAIRACH] [AAL] [HARVARDOXFORD] [CPAC200] [CLEAN] [EXTRA1] [EXTRA2] [EXTRA3]",
```

Comment:
In the future, try not to commit files in which you have only edited whitespaces, unless this was intentional.

Reply:
These changes happen when you run the tests locally. @friesenA, there is a script in `boutiques/tools/python` called `cleanup_tests.sh` that you can run before committing which will revert the changes.